docs/Operators.md

```diff
@@ -1891,6 +1891,8 @@ expect(node, inputs=[x], outputs=[y], name='test_averagepool_3d_default')
   Y = (X - input_mean) / sqrt(input_var + epsilon) * scale + B
   ```
   
+  It should also be noted that in training mode, if `T2` is float16, the mean/var should be cast to float when implementing, because calculation of mean and variance may cause overflow in float16 due to summation.
```

Comment:
did you manually edit this?
This change should be added to the defs.cc file, this way when you generate docs it will automatically pick it up

Reply:
CIs are failing for the same reason...

Reply:
Yes, please update the defs.cc and regenerate the documents. I would change the comment to something like
```
The computation of ReduceMean and ReduceVar uses float to avoid overflow for float16 inputs.
```
Does this mean that it is not required for bfloat16 (only for float16)? Just trying to understand, even if it is an implementation-dependent feature.

Reply:
Regenerated the docs, sorry for the mistake.
@gramalingam for bfloat16 case, since bfloat16 has the same range as float32 (not as precise because its fraction bits are truncated), it does not cause overflow. There could be precision issue with bfloat16, but I think this is not unique to BatchNormalization.